README.md

```diff
@@ -14,14 +14,6 @@ interact with their users' Nimiq addresses.
         - [Checkout](#checkout)
         - [Choose Address](#choose-address)
         - [Sign Transaction](#sign-transaction)
-        - [Signup](#signup)
-        - [Login](#login)
-        - [Onboard](#onboard)
-        - [Logout](#logout)
-        - [Export](#export)
-        - [Change Password](#change-password)
-        - [Add Address](#add-address)
-        - [Rename](#rename)
```

Comment:
I wouldn't remove them from the documentation. State that they are not (yet) publicly available but restricted to the Safe instead. Locally developers can already build against those endpoints.

Reply:
Makes sense to me as well :+1: Maybe put them under a section "wallet management" or something in the documentation, stating that these endpoints will be opened up for third-party use in an upcoming version.